docs/content/configuration/broker.md

```diff
@@ -15,7 +15,8 @@ The broker node uses several of the global configs in [Configuration](../configu
 |Property|Description|Default|
 |--------|-----------|-------|
 |`druid.host`|The host for the current node. This is used to advertise the current processes location as reachable from another node and should generally be specified such that `http://${druid.host}/` could actually talk to this process|InetAddress.getLocalHost().getCanonicalHostName()|
-|`druid.port`|This is the port to actually listen on; unless port mapping is used, this will be the same port as is on `druid.host`|8082|
+|`druid.plainTextPort`|This is the port to actually listen on; unless port mapping is used, this will be the same port as is on `druid.host`|8082 if `druid.server.http.plainText` is set otherwise -1|
+|`druid.tlsPort`|TLS port for HTTPS connector, if `druid.server.http.tls` is set then this config will be used. If `druid.host` contains port then that port will be ignored, this should be a non-negative Integer.|8282 if `druid.server.http.tls` is set otherwise -1|
```

Comment:
I think we don't really need the "otherwise -1" thing. The doc already says the `druid.tlsPort` is only going to be used if `druid.server.http.tls` is set.